### ISSUE TITLE
no-return-wrap crash eslint on empty return statement

### ISSUE THREAD

**Issue by zloirock** [xjamundx/eslint-plugin-promise#49]
```js
promise.then(it => {
    return;
});
```
=>
```
Cannot read property 'type' of null
TypeError: Cannot read property 'type' of null
    at EventEmitter.ReturnStatement (/Users/zloirock/wd/static/node_modules/eslint-plugin-promise/rules/no-return-wrap.js:25:28)
    at emitOne (events.js:101:20)
    at EventEmitter.emit (events.js:188:7)
    at NodeEventGenerator.enterNode (/Users/zloirock/wd/static/node_modules/eslint/lib/util/node-event-generator.js:40:22)
    at CodePathAnalyzer.enterNode (/Users/zloirock/wd/static/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:608:23)
    at CommentEventGenerator.enterNode (/Users/zloirock/wd/static/node_modules/eslint/lib/util/comment-event-generator.js:97:23)
    at Controller.enter (/Users/zloirock/wd/static/node_modules/eslint/lib/eslint.js:925:36)
    at Controller.__execute (/Users/zloirock/wd/static/node_modules/eslint/node_modules/estraverse/estraverse.js:397:31)
    at Controller.traverse (/Users/zloirock/wd/static/node_modules/eslint/node_modules/estraverse/estraverse.js:501:28)
    at Controller.Traverser.controller.traverse (/Users/zloirock/wd/static/node_modules/eslint/lib/util/traverser.js:36:33)
```
Latest versions of `eslint-plugin-promise`, `eslint`, `babel-eslint`

**Comment by xjamundx** [comment]
@zloirock can you take a stab at this?

**Comment by bonesoul** [comment]
it's still crashing eslint for me. is this resolved?

**Comment by leedm777** [comment]
@bonesoul I can confirm that the fix works for me, but it hasn't been published to NPM yet.

**Comment by xjamundx** [comment]
let me see what's up

**Comment by xjamundx** [comment]
published `v3.4.1`

**Comment by leedm777** [comment]
@xjamundx That's some awesome response time. Thanks!
